frontend/src/metabase/components/tree/TreeNode.styled.jsx

```diff
@@ -0,0 +1,73 @@
+import styled, { css } from "styled-components";
+import colors, { lighten } from "metabase/lib/colors";
+import Icon from "metabase/components/Icon";
+
+const TEXT_COLOR_BY_VARIANT = {
+  default: colors["brand"],
+  admin: colors["text-medium"],
+};
+
+const BACKGROUND_COLOR_BY_VARIANT = {
+  default: colors["brand"],
+  admin: colors["accent7"],
+};
```

Comment:
I think referencing colors directly like this upon app instantiation may make whitelabeling/theming not work (not 100% sure about that though). Whitelabeling works by mutating the properties in the `colors` object. Can you instead make these functions so that accessing the `colors` props is lazy?

Reply:
Nice catch, thank you!